CMakeLists.txt

```diff
@@ -844,18 +844,6 @@ if(NANO_GUI OR RAIBLOCKS_GUI)
 
   target_link_libraries(nano_wallet Boost::process rpc node qt)
 
-  if(WIN32)
-    target_link_libraries(nano_wallet Qt5::WinExtras)
```

Comment:
We most likely don't want to remove `Qt5::WinExtras` from the normal `nano_wallet` target.